src/state/bidib_state_setter.c

```diff
@@ -403,7 +403,7 @@ void bidib_state_lc_wait(t_bidib_node_address node_address, t_bidib_peripheral_p
 		peripheral_state->data.wait = (unsigned char) (time & 0x7F);
 	} else {
 		syslog(LOG_ERR, "No peripheral on port 0x%02x 0x%02x configured for node address "
-				       "0x%02x 0x%02x 0x%02x 0x0",
+						"0x%02x 0x%02x 0x%02x 0x0",
```

Comment:
Same alignment 'problem'.
```suggestion
		                "0x%02x 0x%02x 0x%02x 0x0",
```